modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -63,27 +76,41 @@ The two following values are not part of `syslog.h` but are widely used:
 
 Map the log levels or priorities of other logging systems to their nearest match in the preceding list. For example, from link:https://docs.python.org/2.7/library/logging.html#logging-levels[python logging], you can match `CRITICAL` with `crit`, `ERROR` with `err`, and so on.
 
-=== pid
+[horizontal]
+Data type:: keyword
+Example value:: `info`
 
+== pid
 
 The process ID of the logging entity, if available.
 
-=== service
+[horizontal]
+Data type:: keyword
 
+== service
 
 The name of the service associated with the logging entity, if available. For example, syslog's `APP-NAME` and rsyslog's `programname` properties are mapped to the service field.
 
-=== tags
+[horizontal]
+Data type:: keyword
 
+== tags
 
 Optional. An operator-defined list of tags placed on each log by the collector or normalizer. The payload can be a string with whitespace-delimited string tokens or a JSON list of string tokens.
 
-=== file
+[horizontal]
+Data type:: text
 
+== file
 
 The path to the log file from which the collector read this log entry. Normally, this is a path in the `/var/log` file system of a cluster node.
```

Comment:
The path to the log file from which the collector readS this log entry? S missing?

Reply:
Updated. Thanks.